start.go

```diff
@@ -246,11 +240,106 @@ command(s) that get executed on start, edit the args parameter of the spec. See
 // * In runv, shared namespaces multiple containers are located in the same VM which is managed by a runv-daemon.
 // * Any running container can exit in any arbitrary order, the runv-daemon and the VM are existed until the last container of the VM is existed
 
-func startContainer(context *cli.Context, container, address string, config *specs.Spec) int {
-	pid := os.Getpid()
+func createContainer(context *cli.Context, container, address string, config *specs.Spec) error {
 	r := &types.CreateContainerRequest{
 		Id:         container,
+		Runtime:    "runv-create",
 		BundlePath: context.String("bundle"),
+	}
+
+	c := getClient(address)
+	if _, err := c.CreateContainer(netcontext.Background(), r); err != nil {
+		fmt.Printf("error %v\n", err)
+		return err
+	}
+
+	return nil
+}
+
+var startCommand = cli.Command{
+	Name:  "start",
+	Usage: "executes the user defined process in a created container",
+	ArgsUsage: `<container-id>
+
+Where "<container-id>" is your name for the instance of the container that you
+are starting. The name you provide for the container instance must be unique on
+your host.`,
+	Description: `The start command executes the user defined process in a created container`,
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  "console",
+			Usage: "specify the pty slave path for use with the container",
+		},
+		cli.StringFlag{
+			Name:  "pid-file",
+			Usage: "specify the file to write the process id to",
+		},
+		cli.BoolFlag{
+			Name:  "no-pivot",
+			Usage: "[ignore on runv] do not use pivot root to jail process inside rootfs.  This should be used whenever the rootfs is on top of a ramdisk",
+		},
+		cli.BoolFlag{
+			Name:  "detach, d",
+			Usage: "detach from the container's process",
+		},
```

Comment:
Following runc command line, the `pid-file`, `no-pivot` should be flags of `create` command, `detach` should be flag of `run` command, and `console` should be replaced with `--console-socket`(we can do it later in following PR :-) ) and also it's a `create` cmd flag.

`start` command should takes no flags.